### DISCUSSION TITLE
[Support]: Using Authentication with Cloudflare Tunnel

### DISCUSSION THREAD

**Discussion by bobloadmire** [blakeblackshear/frigate#13041] [Ask A Question] [answered]
### Describe the problem you are having

Trying to set this up, tunneling points to https://internalip:8971/ but i'm getting 502 bad gateway. Any advice? works fine locally, just not through tunnel.

### Version

0.14

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Proxmox

### Install method

Docker Compose

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment, marked as answer]
disable tls

**Reply by bobloadmire** [reply]
Thank you!
